themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_nl.properties

```
@@ -1 +1,174 @@
-fullName={0} {1}
\ No newline at end of file
+pageNotFound=Pagina niet gevonden
+forbidden=Verboden
+needAccessRights=U heeft niet de rechten voor dit verzoek. Neem contact op met de administrator.
+invalidRoute={0} is geen valide route.
+actionRequiresIDP=Deze actie vereist een omleiding naar uw identity provider.
+actionNotDefined=Geen actie gedefinieerd
+continue=Doorgaan
+refreshPage=Ververs de pagina
+refresh=Verversen
+done=Klaar
+cancel=Annuleren
+remove=Verwijderen
+update=Wijzigen
+loadingMessage=Account Console laadt ...
+unknownUser=Anoniem
+fullName={0} {1}
+allFieldsRequired=Alle velden zijn verplicht.
+
+selectLocale=Selecteer een taal
+doSignIn=Inloggen
+
+backToAdminConsole=Terug naar admin console
+accountManagementWelcomeMessage=Welkom bij Keycloak account management
+
+# Personal info page
+personalInfoHtmlTitle=Persoonsgegevens
+
+# Device activity page
+signedInDevices=Ingelogde apparaten
+signedInDevicesExplanation=Log alle onbekende apparaten uit.
+signOutWarning=Wilt u alle sessies be\u00EBindigen?
+signOutAllDevices=Alle apparaten uitloggen
+signOutAllDevicesWarning=Deze actie zal alle apparaten die ingelogd zijn met uw account uitloggen, inclusief het apparaat dat u nu gebruikt.
+recentlyUsedDevices=Recentelijk gebruikte apparaten
+recentlyUsedDevicesExplanation=In de afgelopen maand gebruikte apparaten, maar nu niet ingelogd.
+lastAccess=Laatste toegang
+unknownOperatingSystem=Onbekend operating system
```

Perhaps better than using the English form here is to use 'besturingssysteem'.
```suggestion
unknownOperatingSystem=Onbekend besturingssysteem
```